ee/clickhouse/queries/funnels/utils.py

```diff
@@ -13,3 +13,17 @@ def get_funnel_order_class(filter: Filter) -> Type[ClickhouseFunnelBase]:
     elif filter.funnel_order_type == FunnelOrderType.STRICT:
         return ClickhouseFunnelStrict
     return ClickhouseFunnel
+
+
+def get_funnel_order_actor_class(filter: Filter):
```

Comment:
same as posthog/api/person.py `get_funnel_actor_class`?

Reply:
Unfortunately, these two are slightly different. `get_funnel_actor_class` can return the `ClickhouseFunnelTrendsActors` class, and there's some funny behavior in correlations where it can append the `funnel_viz_type` as `TREND` to the filter, and so the wrong actor class is returned.

I spent some time trying to untangle this, but came to the conclusion that I think it's better handled in a different PR